adam-core/src/test/scala/org/bdgenomics/adam/models/SequenceDictionarySuite.scala

```diff
@@ -192,7 +192,7 @@ class SequenceDictionarySuite extends FunSuite {
     toSSD.assertSameDictionary(ssd)
   }
 
-  test("conversion to sam sequence dictionary has correct sort order") {
+  test("conversion to sam sequence dictionary maintains sequence order") {
```

Comment:
I think you are subtly breaking this test. The sort order correction added by #794 is intended for when you have to regenerate the sequence dictionary via an aggregate, which is what you do when you have a Parquet file. It doesn't make sense to me to break (what I consider) the common case, hence my suggestion to toggle the sort order above.